{"name": "test_json_file", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../data/test.json'", "trace": "def test_json_file():\n        print(os.getcwd())\n>       with open('../data/test.json', 'r') as fp:\nE       FileNotFoundError: [Errno 2] No such file or directory: '../data/test.json'\n\ntest/test_file.py:10: FileNotFoundError"}, "attachments": [{"name": "stdout", "source": "2a38db1c-c522-459b-b723-9dbf3e91012f-attachment.txt", "type": "text/plain"}], "start": 1618405247144, "stop": 1618405247145, "uuid": "9afd4d3a-6bfe-410e-952f-b341ff9b5aeb", "historyId": "d50ca8f5466892ba3c198b58ff77ce88", "testCaseId": "052bea2877096ba85b4b1b5378880999", "fullName": "test.test_file#test_json_file", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_file"}, {"name": "host", "value": "avivatangdeMacBook-Pro.local"}, {"name": "thread", "value": "18185-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_file"}]}